src/applications/mhv/medications/containers/LandingPage.jsx

```diff
@@ -7,7 +7,7 @@ import { medicationsUrls } from '../util/constants';
 
 const LandingPage = () => {
   const fullState = useSelector(state => state);
-  const medicationsUrl = !fullState.user.login.currentlyLoggedIn
+  const medicationsUrl = fullState.user.login.currentlyLoggedIn
     ? medicationsUrls.prescriptionsUrl
     : medicationsUrls.medicationsLogin;
```

Comment:
I tried to log in locally and after a successful login, the user isn't taken to the actual list page, instead the user is routed to this `http://localhost:3001/my-health/medications/my-health/medications/?next=loginModal&postLogin=true` url. when it should be this one instead `http://localhost:3001/my-health/medications/prescriptions/?next=loginModal&postLogin=true`

Reply:
fixed